misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-grssignum/src/test/resources/de/metas/camel/externalsystems/grssignum/to_grs/bpartner/30_DispatchRequest.json

```diff
@@ -1,5 +1,5 @@
 {
   "url": "https://grs",
   "authToken": "authToken",
-  "message": "{\"FLAG\":100,\"MKREDID\":\"bp_code\",\"KURZBEZEICHNUNG\":\"name name2 name3\",\"INAKTIV\":0,\"MID\":\"myTenant\",\"NAMENSZUSATZ\":\"name2\",\"ADRESSE 1\":\"address1\",\"ADRESSE 2\":\"address2\",\"ADRESSE 3\":\"address3\",\"ADRESSE 4\":\"address4\",\"PLZ\":\"postal\",\"ORT\":\"city\",\"LANDESCODE\":\"DE\",\"GLN\":\"gln\",\"METASFRESHID\":\"2156425\",\"METASFRESHURL\":\"metasfreshUrl\",\"KREDITORENNR\":1,\"DEBITORRENNR\":2,\"KONTAKTE\":[{\"METASFRESHID\":2205180,\"NACHNAME\":\"contact lastname\",\"VORNAME\":\"contact firstname\",\"ANREDE\":\"greeting\",\"TITEL\":\"title\",\"POSITION\":\"position\",\"EMAIL\":\"email\",\"TELEFON\":\"phone\",\"MOBIL\":\"phone2\",\"FAX\":\"fax\",\"ROLLEN\":[\"role1\"]}]}"
+  "message": "{\"FLAG\":100,\"MKREDID\":\"bp_code\",\"KURZBEZEICHNUNG\":\"name name2 name3\",\"INAKTIV\":0,\"MID\":\"myTenant\",\"NAMENSZUSATZ\":\"name2\",\"ADRESSE 1\":\"address1\",\"ADRESSE 2\":\"address2\",\"ADRESSE 3\":\"address3\",\"ADRESSE 4\":\"address4\",\"PLZ\":\"postal\",\"ORT\":\"city\",\"LANDESCODE\":\"DE\",\"GLN\":\"gln\",\"METASFRESHID\":\"2156425\",\"METASFRESHURL\":\"metasfreshUrl\",\"KREDITORENNR\":1,\"DEBITORRENNR\":2,\"ANHANGORDNER\":\"basePath/directories/bp_code\",\"KONTAKTE\":[{\"METASFRESHID\":2205180,\"NACHNAME\":\"contact lastname\",\"VORNAME\":\"contact firstname\",\"ANREDE\":\"greeting\",\"TITEL\":\"title\",\"POSITION\":\"position\",\"EMAIL\":\"email\",\"TELEFON\":\"phone\",\"MOBIL\":\"phone2\",\"FAX\":\"fax\",\"ROLLEN\":[\"role1\"]}]}"
```

Comment:
here we see a problem (also in your very nice manual test docu btw): 
`ANHANGORDNER` needs to contain just `bp_code` if that's the bpartner's value.
But not the base path. because the base-path can look very differently from the GRS server's POV.
In other words, if e.g. C_BPArtner value=12345, then `ANHANGORDNER` = "12345"